src/libraries/System.Console/tests/ReadKey.cs

```diff
@@ -39,19 +39,33 @@ public static void ConsoleKeyValueCheck()
     }
 
     [Fact]
-    [PlatformSpecific(TestPlatforms.AnyUnix)]
+    [PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.Browser)]
     public void NumberLock_GetUnix_ThrowsPlatformNotSupportedException()
     {
         Assert.Throws<PlatformNotSupportedException>(() => Console.NumberLock);
     }
 
     [Fact]
-    [PlatformSpecific(TestPlatforms.AnyUnix)]
+    [PlatformSpecific(TestPlatforms.Browser)]
+    public void NumberLock_Browser()
+    {
+        Assert.False(Console.NumberLock);
+    }
+
+    [Fact]
+    [PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.Browser)]
     public void CapsLock_GetUnix_ThrowsPlatformNotSupportedException()
     {
         Assert.Throws<PlatformNotSupportedException>(() => Console.CapsLock);
     }
 
+    [Fact]
+    [PlatformSpecific(TestPlatforms.Browser)]
+    public void CapsLock_Browser()
+    {
+        Assert.False(Console.CapsLock);
```

Comment:
Same here.

Reply:
We have `public static bool CapsLock => false;` for both [Android ](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Console/src/System/ConsolePal.Android.cs#L52) and [iOS ](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Console/src/System/ConsolePal.iOS.cs#L57)implementations.